indy_common/config.py

```diff
@@ -102,3 +102,6 @@
 default logging level for node
 '''
 logLevel = logging.INFO
+
+ZMQ_CLIENT_QUEUE_SIZE = 3000  # messages (0 - no limit)
```

Comment:
Why do we set it here and not in plenum?

Reply:
Just for convenience - only one build instead of two: plenum, node with plenum